TACC/Core-Portal
# task/FP-1195: customize onboarding route

## Overview: ##

This PR allows the route that users are directed to after onboarding to be customized. In this PR, the main changes are
* "Access Dashboard" button renamed to "Continue"
* Make route where button goes configurable using _WORKBENCH_SETTINGS (defaults to workbench/dashboard)

## Related Jira tickets: ##

* [FP-1195](https://jira.tacc.utexas.edu/browse/FP-1195)

Also see [Slack discussion with Design](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1631568826032500?thread_ts=1631568780.032400&cid=CQUA4D5KJ)

## Summary of Changes: ##
* "Access Dashboard" button renamed to "Continue"
* Add customize/setting `onboardingSetupCompleteNext` in _WORKBENCH_SETTINGS
* Added unit test to cover where the Continue button goes to (plus some minor refactoring of tests)

## Testing Steps: ##
1. Go to https://cep.dev/workbench/onboarding/setup/ and confirm that Continue leads you to dashboard
2. Change settings (`onboardingSetupCompleteNext` in _WORKBENCH_SETTINGS) and confirm that Continue button works as expected

## UI Photos:
![Screen Shot 2021-09-16 at 2 33 25 PM](https://user-images.githubusercontent.com/8287580/133674624-22f746ac-2295-401e-a2ca-268b01acfb3e.png)
